wporg-gp-translation-events.php

```diff
@@ -163,8 +163,10 @@ function submit_event_ajax() {
 		wp_send_json_error( 'Nonce verification failed' );
 	}
 
-	$title          = isset( $_POST['event_title'] ) ? sanitize_text_field( wp_unslash( $_POST['event_title'] ) ) : '';
-	$description    = isset( $_POST['event_description'] ) ? sanitize_text_field( wp_unslash( $_POST['event_description'] ) ) : '';
+	$title = isset( $_POST['event_title'] ) ? sanitize_text_field( wp_unslash( $_POST['event_title'] ) ) : '';
+	//@codingStandardsIgnoreStart
```

Comment:
Is it an option to call `make_clickable()` already here? And why this style of disabling coding standards vs a `phpcs:ignore` style comment?

Reply:
After some tests, we could decide to call `make_clickable()` here or in the frontend it gives the same result. Do you suggest we rather call it here?